backend/api/views.py

```diff
@@ -169,7 +170,8 @@ def scrimmage_pub_sub_call(red_submission_id, blue_submission_id, red_team_name,
         'name1': str(red_team_name),
         'name2': str(blue_team_name),
         'maps': str(map_ids),
-        'replay': scrimmage_replay
+        'replay': scrimmage_replay,
+        'tourmode': str(tourmode)
```

Comment:
why is this a string and not a naked bool?

Reply:
wanted to keep consistency with everything else being strings (including team ids as strings rather than ints, etc)
if naked bools are better, can change

Reply:
ah i see - the original intent behind the others being strings was that the runner is agnostic about what the data actually is: the only thing it does is plug them into a URL format (with the exception of the maps, which might be more logically a json list but let's not worry about that). on the other hand, the `tourmode` would actually control the runner's behaviour so its "boolness" could be significant here?

Reply:
ahh that makes sense, I'll pass it as real bool
I just realized that game_server checks for it as string anyways, and then uses bool afterwards (line 85) -- but pure bool does seem to make more sense, since the runner does now care about the var

Reply:
(TODO for self -- change line 85 too!)